frontend/src/pages/Registration/registration.jsx

```diff
@@ -1,14 +1,229 @@
+                            name="confirmPassword"
+                            autoComplete="off"
+                            autoCorrect="off"
+                            spellCheck="false"
+                            value={formData.confirmPassword}
+                            onChange={handleChange}
+                            placeholder={'Введіть пароль ще раз'}
+                            className={errors['confirmPassword'] && 'input__error-border-color'}
+                        />
+                        { errors['confirmPassword'] && <p className={"panel--danger-text"}>{ errors["confirmPassword"] }</p> }
+                    </div>
+                    <div>
+                        <PanelBodyTitle title={'Прізвище'} className={'content--text-container__margin'} />
+                        <TextInput
+                            name="lastName"
+                            autoComplete="off"
+                            autoCorrect="off"
+                            spellCheck="false"
+                            value={formData.lastName}
+                            onChange={handleChange}
+                            placeholder={'Введіть ваше прізвище'}
+                            className={errors['lastName'] && 'input__error-border-color'}
+                        />
+                        { errors['lastName'] && <p className={"panel--danger-text"}>{ errors["lastName"] }</p> }
+                    </div>
+                    <div>
+                        <PanelBodyTitle title={'Ім‘я'} className={'content--text-container__margin'} />
+                        <TextInput
+                            name="firstName"
+                            autoComplete="off"
+                            autoCorrect="off"
+                            spellCheck="false"
+                            value={formData.firstName}
+                            onChange={handleChange}
+                            placeholder={'Введіть ваше ім’я'}
+                            className={errors['firstName'] && 'input__error-border-color'}
+                        />
+                        { errors['firstName'] && <p className={"panel--danger-text"}>{ errors["firstName"] }</p> }
+                    </div>
+                    { errors['unexpected'] && <p className={"panel--danger-text"}>{ errors['unexpected'] }</p> }
+                </PanelBody>
+                <PanelNavigation>
+                    <Button
+                        onClick={handleSubmit}
+                        disabled={isLocked}
+                        className={'button__padding panel--button'}
+                    >
+                        Зареєструватися
+                    </Button>
+                </PanelNavigation>
+            </Panel>
+            <div className={"panel--under-panel"}>
+                <span>
+                    Ви вже зареєстровані у нас?
+                </span>
+                <Link className={'text-underline text-bold'} to={'/login'}>
```

Comment:
The login link in the registration page uses '/login', not '/auth/login', causing navigation inconsistency. Update the login link to /auth/login for consistency.

```
<Link className={'text-underline text-bold'} to={'/auth/login'}>
    Увійти
</Link>
```